shared/domains/chelonia/crypto.js

```diff
@@ -255,7 +255,7 @@ export const verifySignature = (inKey: Key | string, data: string, signature: st
     throw new Error('Invalid signature')
   }
 }
-export const encrypt = (inKey: Key | string, data: string): string => {
+export const encrypt = (inKey: Key | string, data: string, ad?: string): string => {
```

Comment:
What is `ad`? Can you please add comments here explaining function parameters, especially `ad` which is very unclear?